zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -169,7 +186,9 @@ function PageWrapper(props) {
     handleSetState({ loading: true });
     fetchHero(props)
       .then(() => {
-        if (props.auth.token) {
+        if (!props.auth.token && !unprotectedRoutes.includes(props.match?.path)) {
+          props.history.push('/session-expired');
```

Comment:
hey @coderatomy, please have a look at this

Reply:
simply paste what I showed you after this line [here](https://github.com/unstructuredstudio/zubhub/blob/master/zubhub_frontend/zubhub/src/views/PageWrapper.jsx#L113)